test/pthread/test_atomic_wait32_notify.c

```diff
@@ -0,0 +1,65 @@
+#include <emscripten/html5.h>
+#include <emscripten/threading.h>
+#include <assert.h>
+
+// Test emscripten_atomic_wait_u32() and emscripten_atomic_notify() functions.
+// Copied from test/wasm_worker/wait64_notify.c. Keep these in sync.
```

Comment:
Maybe add a parallel comment in that file itself, to close the loop? That would avoid the risk of that file being updated without seeing a reminder to update the others.